addons/point_of_sale/static/src/js/barcode_reader.js

```diff
@@ -101,6 +101,17 @@ var BarcodeReader = core.Class.extend({
         }
         this.remote_active = 1;
 
+        $.ajax({
```

Comment:
maybe add "this.iotbox = false" in the init to be sure.

Reply:
yes.. ok